scheduler/kubernetes/upgrade.md

```diff
@@ -45,10 +43,10 @@ This script will also monitor the above operations.
 ### Specify a different Portworx upgrade image
 
 You can invoke the upgrade script with the _-t_ to override the default Portworx image.
-For example below command upgrades Portworx to _portworx/oci-monitor:1.4.2_ image.
+For example below command upgrades Portworx to _portworx/oci-monitor:1.5.1_ image.
 
 ```
-curl -fsL https://install.portworx.com/upgrade | bash -s -- -t 1.4.2
+curl -fsL https://install.portworx.com/upgrade | bash -s -- -t 1.5.1
```

Comment:
Instead of advertising `-t` and `-i` options, we should direct people to different URLs, i.e.:

    https://install.portworx.com/1.6/upgrade